python/tank_vendor/shotgun_authentication/user_impl.py

```diff
@@ -197,7 +197,8 @@ def create_sg_connection(self):
         return _shotgun_instance_factory(
             self.get_host(), session_token=self.get_session_token(),
             http_proxy=self.get_http_proxy(),
-            sg_auth_user=self
+            sg_auth_user=self,
+            connect=False
```

Comment:
Actually, I didn't mean to add this. It's also absent from my code locally. Odd.